PRIVACY.md

```diff
@@ -0,0 +1,48 @@
+# Privacy Policy
+
+*Last updated: 2022-05-30*
+
+Dawn is an Android application licensed under the Apache License 2.0. See [LICENSE](./LICENSE.md) for the full text of this license.
+
+This privacy policy aims to provide information on how Dawn handles your data. This policy may be updated at any time for any reason, and this will be indicated by updating the "last updated" date at the top of the policy.
+
+## Overview
+
+Dawn **doesn't collect any data about you, or your device** and **contains no third party SDKs for adverts, analytics, or telemetry**.
+It does, however, access third party services such as Reddit, which have their own privacy policies and may collect data. See the "Third Party Services" section below.
+
+## Data Stored on Your Device
+
+When you use Dawn, data including (but not limited to) the following may be stored locally on your device.
+
+- App preferences
+- Your logged-in Reddit accounts
+- Information downloaded from Reddit about your account, such as subscribed subreddits/users, voting activity, your inbox contents, etc
+- A cache of previously visited/read pages
+
+## Third Party Services
+
+Dawn is similar to a web browser in that it can be used to access a variety of third party services. These are not operated by the Dawn developers, and therefore you are advised to review the privacy policies of these third party services.
+
+Dawn accesses the Reddit API. This may occur while the app is in use, or periodically in the background to fetch messages and notifications. Reddit is a third party service and outside the scope of this privacy policy. Reddit's privacy policy is available here:
+
+[https://www.redditinc.com/policies/privacy-policy](https://www.redditinc.com/policies/privacy-policy)
+
+Dawn downloads images and other data from third party websites linked to by Reddit posts and comments. This may occur when you click a link, or automatically in the background when viewing a post list (if pre-caching is enabled in the preferences). Also, these services may be accessed directly through the original link, or through an API if one is available. Again, the privacy policies of these third party websites are outside the scope of Dawn’s privacy policy.
```

Comment:
Unfurler makes requests to third-party urls when post is opened without even clicking the link, which exposes device's ip address (but nothing else afaik), and highres version of image may also be downloaded when post is opened